Makefile

```diff
@@ -24,9 +25,15 @@ docker-build:: ## Builds the docker image locally
 			-t $(IMAGE_TAG) $(WORKING_DIR)
 
 docker-run:: ## Runs the docker image locally
```

Comment:
Possible to keep `docker-run` to be docker only and introduce a k8s targets such as `kube-run`, `kube-test` etc.?